src/main/java/org/broadinstitute/hellbender/tools/spark/CreateHadoopBamSplittingIndex.java

```diff
@@ -32,6 +34,7 @@
 @CommandLineProgramProperties(summary = "Create a HadoopBam splitting index as well as a bam index from a bam file.",
         oneLineSummary = "create a hadoop-bam splitting index" ,
         programGroup = SparkProgramGroup.class)
+@DocumentedFeature(groupName= HelpConstants.DOCS_CAT_DATA)
```

Comment:
@vdauwera Yes, we could use the custom tag (or another custom tag) for that, and it would work for online doc. But we'd still have the same problem with command line help output, which (currently) organizes the output by CommandLineProgramGroup and doesn't know anything about Javadoc tags or DocumentedFeature annotations. Of course, we could change that.